system-addon/content-src/asrouter/schemas/message-format.md

```diff
@@ -21,3 +21,18 @@ Field name | Type     | Required | Description | Example / Note
   }
 }
 ```
+
+### HTML subset
+The following tags are allowed in the content of the snippet: `i, b, u, strong, em, br`.
+
+Links cannot be rendered using regular anchor tags because [Fluent does not allow for href attributes](https://github.com/projectfluent/fluent.js/blob/a03d3aa833660f8c620738b26c80e46b1a4edb05/fluent-dom/src/overlay.js#L13). They will be wrapped in custom tags, for example `<cta>link</cta>` and the url will be provided as part of the payload:
+```
+{
+  "links": {
```

Comment:
Can you add the corresponding `text` in this example json so the whole thing is more clear?